src/oscar/core/customisation.py

```diff
@@ -36,99 +38,100 @@ def subfolders(path):
+            "from {app_name}.admin import *  # noqa\n".format(
+                app_name=app_config.name))
 
     logger.info("Creating app config")
-    inherit_app_config(local_app_path, app_package, label)
+    inherit_app_config(local_app_path, local_package_name, app_config)
 
-    # Only create models.py and migrations if it exists in the Oscar app
-    oscar_models_path = join(oscar_app_path, 'models.py')
-    if exists(oscar_models_path):
+    # Only create models.py and migrations if they exist in the Oscar app
+    models_path = join(app_path, 'models.py')
+    if exists(models_path):
         logger.info("Creating models.py")
         create_file(
             join(local_app_path, 'models.py'),
-            "from oscar.apps.%s.models import *  # noqa isort:skip\n" % label)
+            "from {app_name}.models import *  # noqa isort:skip\n".format(
+                app_name=app_config.name))
 
-        migrations_path = 'migrations'
-        source = join(oscar_app_path, migrations_path)
-        if exists(source):
-            logger.info("Creating %s folder", migrations_path)
-            destination = join(local_app_path, migrations_path)
-            shutil.copytree(source, destination)
+        subfolder_path = 'migrations'
+        migrations_path = join(app_path, subfolder_path)
+        if exists(migrations_path):
+            logger.info("Creating %s folder", subfolder_path)
+            local_migrations_path = join(local_app_path, subfolder_path)
+            shutil.copytree(migrations_path, local_migrations_path)
 
     # Final step needs to be done by hand
+    app_config_class_path = "{module_name}.{class_name}".format(
+                            module_name=app_config.__module__,
+                            class_name=type(app_config).__name__)
+    local_app_config_class_path = "{local_package_name}.apps.{class_name}".format(
+                                  local_package_name=local_package_name,
+                                  class_name=type(app_config).__name__)
     msg = (
-        "The final step is to add '%s' to INSTALLED_APPS "
-        "(replacing the equivalent Oscar app). This can be "
-        "achieved using Oscar's get_core_apps function - e.g.:"
-    ) % app_package
+        "The final step is to add '{}' to INSTALLED_APPS, "
+        "replacing the equivalent Oscar app. E.g.:"
+    ).format(local_app_config_class_path)
     snippet = (
         "  # settings.py\n"
         "  ...\n"
         "  INSTALLED_APPS = [\n"
         "      'django.contrib.auth',\n"
         "      ...\n"
         "  ]\n"
-        "  from oscar import get_core_apps\n"
-        "  INSTALLED_APPS = INSTALLED_APPS + get_core_apps(\n"
-        "      ['%s'])"
-    ) % app_package
-    record = "\n%s\n\n%s" % (
+        "  from oscar import OSCAR_CORE_APPS\n"
+        "  INSTALLED_APPS = INSTALLED_APPS + OSCAR_CORE_APPS\n"
+        "  INSTALLED_APPS[INSTALLED_APPS.index('{}')] = '{}'"
```

Comment:
I am not sure about  these instructions. I'd be in favour of `Replace the entry 'oscar.apps.partner' with 'fish_shop.apps.partner' in INSTALLED_APPS` And drop the `OSCAR_CORE_APPS` setting in favour of documentation.